packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -85,6 +106,8 @@ contract Staker is StakerInterface, Ownable, Testable {
             voterStake.unstakeTime = 0;
             votingToken.transfer(msg.sender, tokensToSend);
         }
+
+        emit StakeExecuted(msg.sender, tokensToSend);
```

Comment:
same nit: rename to `ExecutedUnstake`. Include the stakers `voterStake.cumulativeStaked` as well to see how much was left after the unstake.